cluster/shard_mapper.go

```diff
@@ -230,10 +230,21 @@ func (r *RemoteMapper) NextChunk() (chunk interface{}, err error) {
 			}
 			aggValues = append(aggValues, v)
 		}
-		mo.Values = []*tsdb.MapperValue{&tsdb.MapperValue{
-			Value: aggValues,
-			Tags:  mvj[0].Tags,
-		}}
+
+		if mvj[0].Time != 0 {
```

Comment:
Yes, that's right. just need add `Time`. @otoolep 

and there will be another fix very soon following this PR.